lib/index.js

```diff
@@ -4,6 +4,7 @@ import { NativeModules } from 'react-native'
 // const { type, value } = await NativeModules.ShareExtension.data()
 // NativeModules.ShareExtension.close()
 export default {
-  data: () => NativeModules.ReactNativeShareExtension.data(),
-  close: () => NativeModules.ReactNativeShareExtension.close()
+  data: (appGroupId) => NativeModules.ReactNativeShareExtension.data(appGroupId),
+  dataMulti: (appGroupId) => NativeModules.ReactNativeShareExtension.dataMulti(appGroupId),
```

Comment:
Do we need a separate method or can it be supported in the existing `data` method.